homeassistant/components/lg_thinq/entity.py

```diff
@@ -102,14 +102,14 @@ async def async_call_api(
         """Call the given api and handle exception."""
         try:
             await target
-        except ThinQAPIException as exc:
+        except (ThinQAPIException, ValueError) as exc:
             if on_fail_method:
                 on_fail_method()
 
             raise ServiceValidationError(
-                exc.message,
+                exc.message if isinstance(exc, ThinQAPIException) else str(exc),
```

Comment:
I think it would make sense to just have 2 `except`s so you don't have two of these if statements

Reply:
Fixed, Please review.